packages/core/typings/src/embark.d.ts

```diff
@@ -63,6 +63,8 @@ export interface Config {
   reloadConfig(): void;
 }
 
+type ActionCallback = (params: any, cb: (error: any, result: any) => void) => void;
```

Comment:
@emizzle `action` seems to need a different signature than `Callback<any>`; the way `registerActionForEvent` is used it's passed a callback that expects a `params` first-argument (i.e. not an error|null) and a callback as the second argument. However, `ActionCallback` could perhaps be written as:

```suggestion
type ActionCallback = (params: any, Callback<any>) => void;
```